src/Core/src/Core/ISafeAreaPage.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Provides functionality for the Page's SafeAreaInsets that may be changed in the future.
+	/// </summary>
+	/// <remarks>
+	/// This interface is only recognized on the iOS/Mac Catalyst platforms; other platforms will ignore it.
+	/// </remarks>
+	internal interface ISafeAreaPage
```

Comment:
rename this file back to ISafeAreaView2 and make sure to fix all the class that ISafeAreaPage has been added to